packages/web-app/src/components/appli/Entry/SensitiveCaveWarning.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import Alert from '../../common/Alert';
+
+const SensitiveCaveWarning = () => {
+  const { formatMessage } = useIntl();
+  return (
+    <Alert
+      disableMargins
+      severity="warning"
+      title={formatMessage({
+        id: 'Restricted access entrance'
+      })}
+      content={formatMessage({
+        id:
+          "This entrance requires special protection measures. We do not communicate its precise location on Grottocenter (except if you are a Grottocenter's administrator)."
```

Comment:
@urien : Je ne sais pas si c'est une bonne idée d'afficher ici "(except if you are a Grottocenter's administrator)" par rapport à ce que la fede US voulait. En effet il est peut mieux de ne pas trop communiquer sur le fait que certains utilisateurs "privilégiés" peuvent voir cette information non ?

Reply:
D'accord, on peut enlever cette précision